Utils/Dataflow/skel/stage.py

```diff
@@ -27,7 +26,16 @@
 
 
 def process(stage, message):
-    """ Single message processing. """
+    """ Single message processing.
+
+    :param stage: ETL processing stage
+    :type stage: pyDKB.dataflow.stage.ProcessorStage
+    :param message: input message with data to be processed
+    :type message: pyDKB.dataflow.communication.messages.JSONMessage
```

Comment:
Not `AbstractMessage`?

Reply:
No. It is an example of a *real* use-case funcion docs, and in the real use-case one should better specify the exact type of message his (her) function accepts. `AbstractMessage` should be in the placeholder method of `ProcessorStage` (and if it is not there, that's a mistake).

Reply:
Aha, I see.

> `AbstractMessage` should be in the placeholder method of `ProcessorStage` (and if it is not there, that's a mistake).

It's not there because there is no type specified at all - placeholder method's documentation is not completed, but that's an issue for some other PR. `skip_process()` correctly has `AbstractMessage`.